packages/message-backend/src/email/create/templates/partials/paymentRow.hbs

```diff
@@ -0,0 +1,88 @@
+<tr>
+    <td class="email-body" width="100%" cellpadding="0" cellspacing="0">
+        <table class="email-body_inner" align="center" width="414" cellpadding="0" cellspacing="0" role="presentation">
+            <!-- Body content -->
+            <tr>
+                <td class="content-cell">
+                    <div class="f-fallback">
+                        <h1 style="margin-bottom: 0; border-top: 1rem solid #ffffff">{{I18n 'orderConfirmation.receipt.header'}}</h1>
+
+                        <table class="purchase" width="100%" cellpadding="0" cellspacing="0" role="presentation">
+                            <tr>
+                                <td colspan="2">
+                                    <table class="purchase_content" width="100%" cellpadding="0" cellspacing="0">
+                                        {{#each order.items}}
+                                            <tr>
+                                                <td width="80%" class="purchase_item">
+                                                                    <span class="f-fallback" style="font-size:14px; font-weight:bold;">
+                                                                        {{productName}}
+                                                                    </span>
+                                                </td>
+                                                <td class="align-right" width="20%" class="purchase_item">
+                                                                    <span class="f-fallback no-wrap">
+                                                                        {{rowPriceTotal}} €
+                                                                    </span>
+                                                </td>
+                                            </tr>
+
+                                            <tr>
+                                                <td width="80%" class="purchase_item">
+                                                                <span class="f-fallback">
+                                                                    {{I18n 'orderConfirmation.receipt.alv'}} ({{vatPercentage}}%)
```

Comment:
Lets try to always use english (alv -> vat)

Reply:
Good point!